core/src/main/java/inetsoft/web/composer/vs/objects/controller/ClipboardController.java

```diff
@@ -97,7 +64,8 @@ public void copyObject(@Payload CopyVSObjectsEvent event,
                           CommandDispatcher dispatcher,
                           @LinkUri String linkUri) throws Exception
    {
-      copyOrCut(event, headerAccessor, principal, dispatcher, linkUri);
+      clipboardControllerServiceProxy.copyOrCut(runtimeViewsheetRef.getRuntimeId(), event,
+                                                headerAccessor, principal, dispatcher, linkUri);
```

Comment:
The header accessor cannot be transferred across the proxy. You need to use MessageContextHolder on the service side.